Sınıf isim uyuşmazlığı

aykutkardas opened the pull request:
Component in sınıf adı ile render da kullanılan adı arasında uyuşmazlık bulunuyordu.

aykutkardas committed:
Sınıf isim uyuşmazlığı

Component in sınıf adı ile render da kullanılan adı arasında uyuşmazlık bulunuyordu.

omergulcicek commented:
Merhaba Aykut, desteğin için teşekkür ederim.
Dokümanda string ifadeleri Türkçe'ye çevirdim. Onun dışında fonksiyon, element, değişken, component vs her şey orijinalde olduğu gibi İngilizce. Yani burada Hello'yu "Merhaba" yapmak yerine Merhaba componentini Hello olarak düzeltirsen daha doğru olacak, ardından merge edelim :)

aykutkardas committed:
Örneklerdeki Sınıf İsimleri Düzeltmesi

Sınıf isimleri aynı örnek içerisinde Türkçe ve İngilizce olarak bulunuyordu. İngilizce olarak Eşleştirildi.

omergulcicek commented:
Şimdi oldu, merge ettim :+1:
